image_classification/index.html

```diff
@@ -43,6 +43,9 @@
               <label class="btn btn-outline-info custom" name="webnn">
                 <input type="radio" name="backend" id="webnn_gpu" autocomplete="off">WebNN (GPU)
               </label>
+              <label class="btn btn-outline-info custom" name="webnn">
+                <input type="radio" name="backend" id="webnn_npu" autocomplete="off">WebNN (NPU)
```

Comment:
Shall we disable NPU if the test browser doesn't support it? Similar to the logic of disabling the `Wasm (CPU)`.

Reply:
Good idea, done.